BREAKING_CHANGES.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased
+
+*   When using a commit message that matches **both** `*-version-bump-message` and `no-bump-message`, there is no increment for that commit. In other words, `no-bump-message` now takes precedence over `*-version-bump-message`.
```

Comment:
Perhaps we should mention this in the documentation for `*-bump-message` as well?

Reply:
Done!

Reply:
Awesome!